nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h

```diff
@@ -356,6 +357,27 @@ struct nrf_wifi_fmac_priv_def {
 #endif /* CONFIG_NRF700X_STA_MODE */
 };
 
+/**
+ * @brief Structure to hold raw tx packet information.
+ *
+ * This structure holds the information sent by higher
+ * layers to transmit a raw frame.
+ */
+struct raw_tx_data {
+	/** magic number */
+	unsigned int magic_num;
+	/** Data rate of the packet */
+	unsigned char data_rate;
+	/** Packet length */
+	unsigned short packet_length;
+	/** Mode describing if packet is VHT, HT, HE or Legacy */
+	unsigned char tx_mode;
+	/** Wi-Fi access category mapping for packet */
```

Comment:
Same comment here (above giving reference to corresponding enum)

Reply:
done